packages/components/src/styles/footnotes/footnotes.stories.ts

```diff
@@ -55,39 +52,42 @@ export const Default = {
 };
 
 /**
- * Use the class with a `ul` element to display the footnotes as an unordered list.
+ * Use the alternative html elements for alternative appearances:
+ *
+ * - unnumbered: use a `ul` element to create an unnumbered list.
+ * - numbered: use an `ol` element to create a numbered list.
+ * Usually start with "1", use the html `start` attribute to set the starting number of the list.
  */
 
-export const UnorderedLists = {
-  parameters: { controls: { include: [] } },
-
-  render: (args: any) => {
-    return generateTemplate({
-      options: {
-        templateContent: '<div class="sd-container"><ul class="%CLASSES%">%SLOT%</ul></div?'
-      },
-      args
-    });
-  }
+export const Variants = {
```

Comment:
For the variants can we star with the numbered variant as it is default one?
And use 1, 2, 3 instead of 150, 151, 152?